Libplanet/IBencodexable.cs

```diff
@@ -0,0 +1,6 @@
+namespace Libplanet
+{
+    public interface IBencodexable
```

Comment:
What's this?

Reply:
I added it to except that some classes become csharp object, during `BencodexFormatter<T>.Deserialize`,`FromBencodexValue`;

But it isn't needed anymore, so I will remove it.